packages/standalone-analyzer/src/options.ts

```diff
@@ -0,0 +1,42 @@
+import { type Output } from "@codecov/bundler-plugin-core";
+
+/** Configuration options for the standalone analyzer. */
+export interface StandaloneOptions {
+  /**
+   * Asynchronous function to handle dry run operations.
+   *
+   * This function is called when a dry run is performed. By default, it logs the report as a JSON
+   * string using `console.log`.
+   *
+   * @returns {Promise<void>} A promise that resolves when the dry run operation is complete.
+   */
+  dryRunner?: (report: Output) => Promise<void>;
```

Comment:
why do we expose this?

I'd rather avoid this complexity being part of the public API, and adding it if someone asks for it later.

Reply:
We do provide the ability to enable `dryRun` in all the other plugins, so users can test things out to ensure that they've configured everything and that the stats will output something.

Reply:
Yeah enabling `dryRun` is fine, I don't like exposing how dry run works as public API, should by purely implementation detail in my eyes.

Reply:
ahh ok ok makes sense